Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2760,6 +2760,17 @@ def return_results(results):
         demisto.results(None)
         return
 
+    if isinstance(results, list):
+        if not results:
+            # demisto.results([]) is invalid
+            demisto.results({})
+            return
+
+        elif isinstance(results[0], CommandResults):  # other cases should be handled by default case
```

Comment:
check in single condition if results exists, is a list, has length > 0 and the first item is of type CommandResults